src/test/java/org/opentripplanner/common/geometry/SubgraphTest.java

```diff
@@ -0,0 +1,54 @@
+package org.opentripplanner.common.geometry;
+
+import org.junit.Test;
+import org.opentripplanner.common.geometry.Subgraph;
+import org.opentripplanner.model.Stop;
+import org.opentripplanner.routing.bike_park.BikePark;
+import org.opentripplanner.routing.graph.Graph;
+import org.opentripplanner.routing.graph.Vertex;
+import org.opentripplanner.routing.vertextype.BikeParkVertex;
+import org.opentripplanner.routing.vertextype.TransitVertex;
+import org.opentripplanner.routing.vertextype.PatternStopVertex;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+
+public class SubgraphTest {
+
+  @Test
+  public void vertex() {
+    // Arrange
+    final Subgraph subgraph = new Subgraph();
+    final BikeParkVertex vertex = new BikeParkVertex(new Graph(),  new BikePark());
```

Comment:
Is there a reason this is a `BikeParkVertex` instead of the superclass `Vertex`? Maybe it doesn't matter?

Reply:
Vertex is an abstract class.

Reply:
Whoops - my bad